backend/routes/api/images.ts

```diff
@@ -0,0 +1,100 @@
+import { Router, Request, Response, NextFunction } from "express";
+import { check } from "express-validator";
+import { handleValidationErrors } from "../../utils/validation.js";
+
+const router = Router();
+
+import bcrypt from "bcryptjs";
+import { setTokenCookie, restoreUser, requireAuth } from "../../utils/auth.js";
+import { prisma } from "../../dbclient.js";
+
+
+
+// ! Delete spot by imageId
+
+
+router.delete('/api/spot-images/:imageId', async (req, res) => {
```

Comment:
this route should be scoped as it should be imported by routes/api.ts; `delete("/spot-images/:imageId")`